packages/core/saved-objects/core-saved-objects-server/index.ts

```diff
@@ -52,6 +52,7 @@ export type {
   SavedObjectsExportablePredicate,
 } from './src/saved_objects_management';
 export type { SavedObjectStatusMeta } from './src/saved_objects_status';
+export { MAIN_SAVED_OBJECT_INDEX } from './src/saved_objects_index_pattern';
```

Comment:
Do we really need this to be exported from the public package? I'd rather keep this internal (and in one of our internal package), especially given we already have an API for core-externals to retrieve this info at runtime: 

https://github.com/elastic/kibana/blob/47d1a0e50fcd1ab155308c497b6f978d0d78b9fd/packages/core/saved-objects/core-saved-objects-server/src/contracts.ts#L136-L139

Reply:
The `'.kibana'` string is widely hardcoded across the codebase, specially in tests.
I wanted to factorise this and centralise this knowledge somehow.
The end goal being discoverability and also having a centralised place to define the names of the SO indices.

If there's a test that manually deletes `.kibana` index, I would like to replace that with a list of all SO indices.
This list could be exposed by core (see [my draft PR](https://github.com/elastic/kibana/pull/154444/files#diff-ac5793b14a45f7408c772835af2d270ec25ec0117ade92f192df98385d73961e)) so that when we decide that we add a new SO index, the tests are transparently taking it into account.

Reply:
After looking at the external usages of `.kibana` (e.g in FTR / es-tools), I think this export is fine.

Ideally it would be exported from an utility package and not from the public type one, but this can be changed later as a follow-up